gamemodes/horde/gamemode/status/buff/sv_warden_aura.lua

```diff
@@ -1,8 +1,11 @@
 local entmeta = FindMetaTable("Entity")
 local plymeta = FindMetaTable("Player")
 
+util.AddNetworkString("AddNewAuraRing")
+
```

Comment:
nitpick, you dont need a newline here

Reply:
I forgot above those, they aren't even needed at all anymore